core/lib/config/src/configs/da_client/eigen.rs

```diff
@@ -28,11 +28,13 @@ pub struct DisperserConfig {
     pub eth_confirmation_depth: i32,
     pub eigenda_eth_rpc: String,
     pub eigenda_svc_manager_address: String,
-    pub blob_size_limit: u64,
+    pub blob_size_limit: u32,
```

Comment:
why this change?

Reply:
It is needed to call `KZG::setup`, since it receives a u32, and it would not be correct to just downcast it.